.eslintrc.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
     'import/extensions': ['error', 'always', {
       ignorePackages: true,
     }],
+    'no-mixed-operators': ['error', {
+      allowSamePrecedence: true,
+    }],
```

Comment:
Happy to revert, but this lint rule made the following arithmetic throw a lint error:

```
const someValue = a + b - c;
```

You _could_ add parentheses to that statement, but adding parentheses there would change the order of operations. You can add them in such a way that it doesn't matter too, but then you're just adding parens for the sake of making lint happy which makes the code uglier. In the real-world example I had, I had something like

```
const value = (x / 2) + (y / 2) - constant;
```

Adding extra parens to this would make it way harder to read imo:

```
const value = ((x / 2) + (y / 2)) - constant;
```

Reply:
🚀